tests/input/pawn/pawn_amxmodx.sma

```diff
@@ -0,0 +1,35 @@
+#include <amxmodx>
+
+#define COOL_MACRO( %1 , %2 ) ( %1 + %2 + 1 )
+
+#define SXO( % 1 ,%2) 1 < %1 || %2 > 2
```

Comment:
As "% 1" (with the space) is invalid and won't compile, then I'm against "fixing" it.  In other words, it should be "% 1" in the output, too.